### DISCUSSION TITLE
nws ontology and where weather-gov-api fits

### DISCUSSION THREAD

**Discussion by vigouredelaruse** [weather-gov/api#640] [Q&A] [answered]
sorry for the google fail but does the nws publish a metadata ontology if you will that
* describes the taxonomy of weather datasets
* equivalent to this https://developer.ap.org/ap-metadata-services/
* depicts where weather-gov-api fits in any such domain specific ontology

thanks

**Comment by dwhitemv25** [comment, marked as answer]
Are you looking for a catalog of all the datasets NWS generates? That has a REALLY, REALLY big answer. 

This forum discusses the NWS API (JSON), whose description is here:

https://www.weather.gov/documentation/services-web-api

For a list of NWS products, this is a decent (but not complete) starting point:

https://www.nco.ncep.noaa.gov/pmb/products/

It would help to know what data points you are looking for.

**Reply by vigouredelaruse** [reply]
@dwhitemv25 thanks for the jumpoff point

i'm building an earth observation tool specific to flood modelling and risk analysis. it's implemented as a multitenant cms suitable for ingesting observations and historicals from ogc and other datasets

i'm doing the information architecture for the part of the user interface like this in qgis
![image](https://github.com/weather-gov/api/assets/31052867/d920043d-ed22-4e1b-952a-9ba0c805c5f1)

i have to allow users to add to a prepopulated list of dataset providers and formats that the tool can ingest, display on a map and route to workflow orchestrations - 

so a user picks an area of interest and nominates some processes to apply - get sar images, get gridded precip, get evapotranspiration, get dem, run model, compare against sar inundation map

and so i'm assembling the taxonomy graph for all those things and that's why i'm asking
thanks

**Reply by dwhitemv25** [reply]
If you're looking for GIS-oriented datasets specifically, try the services described at these sites:

* https://www.weather.gov/gis/WebServices - General description and links for NWS GIS resources

* https://idpgis.ncep.noaa.gov/arcgis/rest/services - ArcGIS REST server providing many NWS & NOAA datasets, including the NDFD data that backs the API. QGIS has a good ArcGIS connector built-in.

These datasets are well documented and available in formats better suited for the types of analysis you describe.

**Reply by sullynole** [reply]
@vigouredelaruse may not be what you're looking for, but the National Water Center is doing a lot of work with the national water model and GIS services for those data: https://water.noaa.gov

**Reply by vigouredelaruse** [reply]
thanks all - @sullynole as someone who didn't take geophys whenever those who did answer my questions it fully behooves me to pursue the avenue from the domain expert :)
thanks
